engines_json/epfl.json

```diff
@@ -1,5 +1,4 @@
 {
```

Comment:
you need to add a `title`

Reply:
Done. https://github.com/grolimur/zotero-lookup-engines/commit/0b3d12cc50303834151233e65d9ab7b7f66253c0

Reply:
Done.